API/Controllers/ProjectController.cs

```diff
@@ -56,7 +57,15 @@ public class ProjectController : ControllerBase
         /// <param name="userService">The user service which is used to communicate with the logic layer.</param>
         /// <param name="fileService">The file service which is used to communicate with the logic layer.</param>
         /// <param name="mapper">The mapper which is used to convert the resources to the models to the resource results.</param>
-        public ProjectController(IProjectService projectService, IUserService userService, IFileService fileService, IMapper mapper, IFileUploader fileUploader)
+        /// <param name="authorizationHelper">The authorization helper which is used to communicate with the authorization helper class.</param>
+        /// <param name="userProjectService">The user project service is responsible for users that are following / liking projects.</param>
+        public ProjectController(IProjectService projectService,
+                                 IUserService userService,
+                                 IMapper mapper,
+                                 IAuthorizationHelper authorizationHelper,
+                                 IFileService fileService,
+                                 IFileUploader fileUploader,
```

Comment:
"Parameter 'fileUploader' has no matching param tag in the XML comment for 'ProjectController.ProjectController(IProjectService, IUserService, IMapper, IAuthorizationHelper, IFileService, IFileUploader, IUserProjectService)' (but other parameters do)
"